docs/features/operators.md

```diff
@@ -0,0 +1,206 @@
+
+When the inputs are ints it will return an int which is the mathematical addition of the two numbers:
+```js
+let val = 1 + 2 // 3
+```
+
+When the inputs are floats it will return a float which is the mathematical addidtion of the two numbers: 
+```js
+let val = 1.5 + 2.5 // 4.0
+```
+
+When the inputs are both strings it will return a string which is the concatination of both of the strings:
+```js
+let val = "hel" + "llo" // "hello"
+```
+
+When one of the inputs are a string and another is a char it will take the string version of the char and concatenate them:
+```js
+let val = \{h} + "ello" // "hello"
+```
+
+### `SUBTRACT`
+The SUBTRACT operator (`-`) can be used on ints and floats as a sign to signal a number as negative or as a mathematical expression
+
+When the inputs are both ints it will return an int which is the mathematical subtraction of the two numbers:
+```js
+let val = 2 - 1 // 1
+
+let val1 = -1 // -1
+```
+
+When the inputs are both floats it will return a float which is the mathematical subtraction of the two numbers:
+```js
+let val = 2.5 - 0.5 // 2.0
+
+let val1 = -1.5 // -1.5
+```
+
+### `MULTIPLY`
+The MULTIPLY operator (`*`) can be used on ints and floats as a mathematical expression
+
+When the inputs are both ints it will return an int which is the mathematical multiplication of the two numbers:
+```js
+let val = 2 * 3 // 6
+```
+
+When the inputs are both floats it will return a float which is the mathematical multiplication of the two numbers:
+```js
+let val = 2.5 * 0.5 // 1.25
+```
+
+### `DIVIDE`
+The DIVIDE operator (`/`) can be used on ints and floats as a mathematical expression
+
+When the inputs are both ints it will return an int which is the mathematical division of the two numbers and it will truncate the number:
+```js
+let val = 7 / 4 // 3
+```
+
+When the inputs are both floats it will return a float which is the mathematical division of the two numbers:
+```js
+let val = 2.5 / 0.5 // 5.0
+```
+
+### `SHIFTL` and `SHIFTR`
+The SHIFTL (`>>`) and SHIFTR (`<<`) operators take in two ints and will bit shift the int on the left the amount of times as the number on the right:
+```js
+let val = 1 << 3 // 0100 from 0001 or 4
+
+let val1 = 8 >> 2 // 0010 from 1000 or 2
+```
+
+### `IN`
+The IN operator (`in`) can be used on values in lists, chars, and strings to measure whether a value is in them.
+
+When the values given are a list and a item whose type is type of the values in the list it will return `true` if the item is in the list:
+```js
+let val = 1 in [1, 2, 3, 4] // true
+```
+
+When the values given are a char and a string or a string and a string it will return true if the char or the string is inside the string:
+```js
+let val = \{e} in "hello" // true
+
+let val1 = "hi" in "hello" // false
+```
+
+### `MODULO`
+The MODULO operator (`%`) can be used to get the remainder of a division between ints or floats
```

Comment:
Module and remainder are different when it comes to negative numbers. Most programming languages use `%` for remainder, but Python and N use it for modulo

Reply:
C++ doesn't have a modulo operator ;)

Reply:
What should I do then, i think it is just fine as is